docs/gettingstarted.md

```diff
@@ -0,0 +1,99 @@
+<!--[metadata]>
++++
+title = "Getting Started"
+description = "Getting started with Notary"
+keywords = ["docker, notary, trust, image, signing, collection, cli"]
+[menu.main]
+parent="mn_notary"
++++
+<![end-metadata]-->
+
+# Getting Started
+
+On this page you create a trusted collection of data using Notary, add data, and publish it to a remote Notary server.
+
+## Prerequisites
+
+Make sure you have already
+[installed Notary](install.md) and are running a [Notary Server](notary-server.md).
+
+## Step 1: Initialization
+
+In order to be able to add content to a trusted collection, you first need to initialize it.
+
+```
+$ notary init example.com/collection
+
+No root keys found. Generating a new root key...
+You are about to create a new root signing key passphrase. This passphrase
+will be used to protect the most sensitive key in your signing system. Please
+choose a long, complex passphrase and be careful to keep the password and the
+key file itself secure and backed up. It is highly recommended that you use a
+password manager to generate the passphrase and keep it safe. There will be no
+way to recover this key. You can find the key in your config directory.
+Enter passphrase for new root key with ID 1f54328:
+Repeat passphrase for new root key with ID 1f54328:
+Enter passphrase for new targets key with ID 1df39fc (example.com/collection):
+Repeat passphrase for new targets key with ID 1df39fc (example.com/collection):
+```
+
+This command initializes a repository named `example.com/collection`, creating a new root key, if one does not exist, and new targets/snapshots keys for this particular collection.
```

Comment:
It might be worth mentioning how the second passphrase is used for both the snapshot and targets keys here?  I'm also curious if we should change "snapshots" to "snapshot" to match the key `ROLE` described below